dev-packages/node-integration-tests/suites/express/without-tracing/test.ts

```diff
@@ -0,0 +1,50 @@
+import { cleanupChildProcesses, createRunner } from '../../../utils/runner';
+
+afterAll(() => {
+  cleanupChildProcesses();
+});
+
+test('correctly applies isolation scope even without tracing', async () => {
+  const runner = createRunner(__dirname, 'server.ts')
+    .ignore('session', 'sessions')
+    .expect({
+      event: {
+        tags: {
+          global: 'tag',
+          'isolation-scope': 'tag',
+          'isolation-scope-1': '1',
+          // We can't properly test non-existance of fields here, so we cast this to a string to test it here
```

Comment:
small nitpick
```suggestion
          // We can't properly test non-existent of fields here, so we cast this to a string to test it here
```

Reply:
Haha, looking this up, neither of these is correct - it should be:

```suggestion
          // We can't properly test non-existence of fields here, so we cast this to a string to test it here
```